src/main/java/uk/gov/companieshouse/acsp/models/filing/ACSP.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.companieshouse.acsp.models.filing;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+@JsonIgnoreProperties(ignoreUnknown = true)
+public final class ACSP {
+
+    @JsonProperty("payment_reference")
+    private String paymentReference;
+    @JsonProperty("payment_method")
+    private String paymentMethod;
+
+    @JsonProperty("correspondence_address")
+    private Address correspondenceAddress1;
```

Comment:
any reason we named this as correspondenceAddress1 instead of correspondenceAddress ?

Reply:
It was causing an issue with dto to dao mapper.. But I will check again.

Reply:
fixed it now